content/1_General/USACO_FAQs.mdx

```diff
@@ -238,6 +238,73 @@ or by googling. Answers to additional FAQs can be found
+
+## Q: How do I initiate or begin a contest?
+
+Once you've registered and logged into your USACO account, check the website during contest weekends for instructions on starting the contest. Usually, there will be a button or link on the homepage indicating the start of the contest. Click on it to initiate your personal contest timer and access the contest problems.
+
+## Q: Is collaborative work allowed?
+
+Collaborating with others during the contest is strictly prohibited to maintain fairness and integrity. USACO contests are designed to assess individual problem-solving skills, and allowing collaboration could provide an unfair advantage. However, outside of the contest environment, students are encouraged to study and prepare together, sharing knowledge and techniques to improve their skills collectively.
+
+## Q: Is participation on a school team mandatory?
+
+No, participating on a school team is not mandatory for USACO contests. Competitors can participate as individuals, and teamwork during the contest itself is not allowed.
+
+## Q: Are there any awards or recognitions for participation?
+
+While USACO does not offer physical prizes, advancing through the difficult divisions and achieving high scores is an accomplishment worth noting. Recognition is given by listing the names of advancing participants in the contest results, which can be a notable achievement to include on college applications and resumes. Additionally, the skills and knowledge gained through participation can be valuable assets in academic and professional pursuits.
+
+## Q: What is the scoring system for contests?
+
+USACO contests are scored out of 1000 points, with each problem weighted equally. Solutions are evaluated against a set of test cases, and points are awarded based on the number of correct test cases passed.
+
+## Q: Are there any penalties for performing poorly?
+
+No, there are no penalties for performing poorly in USACO contests. Participating in the contest provides valuable learning experiences regardless of your performance.
+
+## Q: Is there a possibility of being moved down a division?
+
+USACO follows a promotion-based system, where participants advance to higher divisions based on their performance in contests. Once promoted to a higher division, participants remain in that division unless they advance further. There is no demotion from a division once achieved, providing a sense of progression and achievement for participants.
+
+## Q: Can I participate in multiple divisions concurrently?
+
+Yes, with a perfect score in your current division, you can be immediately promoted to the next division and participate in its contest during the same weekend.
+
+## Q: What types of problems can I expect in USACO?
```

Comment:
I don't think this section is useful, remove. Just check past problems.

Reply:
Done.